spring-cloud-skipper/src/main/resources/org/springframework/cloud/skipper/io/cf-template.yml

```diff
@@ -0,0 +1,9 @@
+apiVersion: skipper.spring.io/v1
```

Comment:
Btw, where this template is used? Can't find references to it as I can with `generic-template.yml`.